frontend/public/locales/en/public.json

```diff
@@ -709,6 +709,7 @@
   "VolumeSnapshotClass": "VolumeSnapshotClass",
   "VolumeSnapshotContent": "VolumeSnapshotContent",
   "VolumeSnapshotContents": "VolumeSnapshotContents",
+  "Select device type": "Select device type",
```

Comment:
why change in this file ?
You dont require this.

Reply:
Ah! it is actaully set Volume Type, i have to run yarn i18n